### ISSUE TITLE
The update does not show up in Julia

### ISSUE THREAD

**Issue by linwaytin** [sswatson/AsyPlots.jl#6]
Hi, I know you just release a new update `0.2.2`.
Somehow I still cannot see it in julia's `Pkg`.
Is that normal?

**Comment by sswatson** [comment]
Yes, tagging creates a PR on the registry which then has to be merged. You can skip the wait by doing 

```
]add AsyPlots#master
```

**Comment by linwaytin** [comment]
Ok got it.
I hope it can be merged soon.

**Comment by linwaytin** [comment]
I checked the [registry](https://github.com/JuliaRegistries/General) but there seems no PR. 
Is this related to the new TagBot (the PR in this repo)?
